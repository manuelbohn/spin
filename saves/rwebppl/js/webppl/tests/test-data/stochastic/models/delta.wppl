var model = function() {
  return delta(42);
};
