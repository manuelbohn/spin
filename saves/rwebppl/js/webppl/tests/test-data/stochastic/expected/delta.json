{
    "hist": {
        "42": 1
    }
}
